addon/components/breadcrumbs.js

```diff
@@ -21,17 +21,26 @@ export default Component.extend({
    */
 
   layout,
-  tagName: 'ul',
+  tagName: '',
+  element: null,
+  containerData: null,
 
   /**
-   * Hooks
+   * Actions
    */
 
-  didInsertElement () {
-    this.get('breadcrumbsService.instances').addObject(this)
+  handleInsert (element) {
+    this.set('containerData', {
+      element,
+      itemClass: this.itemClass,
+      linkClass: this.linkClass
+    })
+
+    this.breadcrumbsService.registerContainer(this.containerData)
   },
 
-  willDestroyElement () {
-    this.get('breadcrumbsService.instances').removeObject(this)
+  handleDestroy () {
+    this.breadcrumbsService.unregisterContainer(this.containerData)
+    this.set('containerData', null)
```

Comment:
Wonder if this is necessary? Isn't the ref to `containerData` removed when the component gets destroyed?

Reply:
No it probably isn't necessary